public/pages/CreateTrigger/containers/CreateTrigger/utils/triggerToFormik.js

```diff
@@ -81,3 +164,51 @@ export function triggerToFormik(trigger, monitor) {
     },
   };
 }
+
+export function getAggregationTriggerConditions(condition) {
+  const triggerConditions = segmentArray(condition.script.source, 4).map((conditionArray) =>
```

Comment:
I believe in `CreateTrigger`, the `triggerConditions` are being formed with the `and/or` condition starting from subsequent condition entries but this `segmentArray` would parse it starting from the first one. We might want to ensure that behavior is consistent here otherwise we could get some strange edge cases.